Sources/Valkey/Connection/ValkeyConnection.swift

```diff
@@ -261,10 +282,13 @@ public final actor ValkeyConnection: ValkeyConnectionProtocol, Sendable {
             }
         }
 
+
+
         let future: EventLoopFuture<Channel>
         switch address.value {
         case .hostname(let host, let port):
-            future = connect.connect(host: host, port: port)
+            let socketAddress = try! SocketAddress(ipAddress: host, port: port)
+            future = connect.connect(to: socketAddress)
```

Comment:
Can see why you did this, but can we keep it to another PR. It would be good to still support hostnames.